src/pages/office/components/debrief-witnessed/debrief-witnessed.ts

```diff
@@ -17,20 +31,33 @@ export class DebriefWitnessedComponent implements OnChanges {
   debriefWitnessedChange = new EventEmitter<boolean>();
 
   private formControl: FormControl;
+  private fieldName: string = 'debriefWitnessed';
+  constructor(private outcomeBehaviourProvider: OutcomeBehaviourMapProvider) { }
 
   ngOnChanges(): void {
     if (!this.formControl) {
-      this.formControl = new FormControl(null, [Validators.required]);
-      this.formGroup.addControl('debriefWitnessed', this.formControl);
+      this.formControl = new FormControl(null);
+      this.formGroup.addControl(this.fieldName, this.formControl);
+    }
+    const visibilityType = this.outcomeBehaviourProvider.getVisibilityType(this.outcome, this.fieldName);
+
+    if (visibilityType === VisibilityType.NotVisible) {
+      this.formGroup.get(this.fieldName).clearValidators();
+    } else {
+      this.formGroup.get(this.fieldName).setValidators([Validators.required]);
     }
     if (this.debriefWitnessed !== null) {
-      this.formControl.patchValue(this.debriefWitnessed ? 'Yes' : 'No');
+      this.formControl.patchValue(this.debriefWitnessed
+        ? ValidWitnessedValues.YES : ValidWitnessedValues.NO);
+    } else {
+      this.formControl.patchValue(null);
     }
   }
 
   debriefWitnessedChanged(debriefWitnessedFormValue: string): void {
     if (this.formControl.valid) {
-      this.debriefWitnessedChange.emit(debriefWitnessedFormValue === 'Yes' ? true : false);
+      this.debriefWitnessedChange.emit(
+        debriefWitnessedFormValue === ValidWitnessedValues.YES ? true : false);
```

Comment:
Same idea, emit the outcome of the expression

Reply:
changed.